src/lib/annotations/image/ImageAnnotator.js

```diff
@@ -85,6 +85,8 @@ class ImageAnnotator extends Annotator {
      * @return {AnnotationThread} Created annotation thread
      */
     createAnnotationThread(annotations, location, type) {
+        this.emit('annotationmodeexit');
```

Comment:
I already pass along w/e mode was entered, I can make sure to pass along what mode was exited as well